pkg/p2p/libp2p/libp2p.go

```diff
@@ -155,7 +155,12 @@ func New(ctx context.Context, signer beecrypto.Signer, networkID uint64, overlay
 		return nil, fmt.Errorf("autonat: %w", err)
 	}
 
-	handshakeService, err := handshake.New(overlay, h.ID(), signer, networkID, o.Logger)
+	underlays, err := buildFullMAs(h.Addrs(), h.ID())
+	if err != nil {
+		return nil, fmt.Errorf("could not build host multiaddresses: %w", err)
```

Comment:
We do not need so long message, it could be just `"build host multiaddresses: %w"`. "Could not" is redundant given that it is an error.